cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -2745,10 +2769,10 @@ private module Stage4 {
 
   bindingset[node, cc, config]
   private LocalCc getLocalCc(NodeEx node, Cc cc, Configuration config) {
-    localFlowEntry(node, _, config) and
```

Comment:
Post-merge drive-by comment: With this removed we no longer need the `config` column. PR: https://github.com/github/codeql/pull/8694
The removed conjunct used to be a needed piece of magic, but the code has changed a lot since it was introduced, so it makes sense that it has become superfluous.

Reply:
Thanks for fixing.